_posts/2020-02-03-reviving-sandstorm.md

```diff
@@ -68,8 +68,8 @@ committed to spending what time I can making Sandstorm's vision a
 reality. If you want to help, join us in the #sandstorm IRC channel
 on libera.chat, and sign up for the [sandstorm-dev mailing list][8].
 
-[1]: https://sandstorm.io/news/2017-02-06-sandstorm-returning-to-community-roots
-[2]: https://sandstorm.io/news/2017-10-28-i18n
+[1]: https://sandstorm.org/news/2017-02-06-sandstorm-returning-to-community-roots
+[2]: https://sandstorm.org/news/2017-10-28-i18n
```

Comment:
These can be local links, see: https://github.com/sandstorm-org/sandstorm-website/blob/a146582e61ef0b0a5deab335368e80037f779ced/_posts/2023-11-03-from-io-to-org.md?plain=1#L79